python_scripts/parameter_tuning_grid_search.py

```diff
@@ -65,7 +58,15 @@
 categorical_columns = categorical_columns_selector(data)
 
 # %% [markdown]
-# Then we build our ordinal encoder, giving it the known categories.
+# Here we will use a tree-based model as a predictor. That means:
```

Comment:
To avoid `predictor`.

```suggestion
# Here we will use a tree-based model as a classifier
# (i.e. `HistGradientBoostingClassifier`). That means:
```